### PR TITLE
feat: css loader

### PR DESCRIPTION
**Pull request by gcofficial**

### CODE REVIEW

File: src/components/grid/BuySell.vue [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 <template lang='pug'>
 .buySell
+  CSSLoader
```
Comment (xerox-h20) [inline src/components/grid/BuySell.vue]: CSSLoader is added unconditionally and is displayed irrespective of client-server communication.

Comment (gcofficial) [reply src/components/grid/BuySell.vue]: Fixed

File: src/components/grid/BuySell.vue [hunk 2]
```diff
@@ -138,12 +140,21 @@ export default {
...
+      .then(
+        () => {
+          this.isDone = true;
+          this.amount = '';
+          this.total = 0;
+          this.open = false;
+          this.loading = false;
+        }
+      )
+      .catch(
```
Comment (xerox-h20) [inline src/components/grid/BuySell.vue]: If the request fails, the loader doesn't stop. See screenshot.

**Comment by gcofficial** [comment]
Fixed

File: src/components/grid/History.vue [hunk 3]
```diff
@@ -47,7 +57,18 @@ export default {
...
     getApiRequest() {
-      this.getTradeHistory();
+      this.loading = true;
+      this.getTradeHistory()
+        .then(
+          () => {
+            this.loading = false;
+          }
+        )
+        .cath(
```
Comment (xerox-h20) [inline src/components/grid/History.vue]: typo in 'catch'

File: src/components/grid/OrderBook.vue [hunk 4]
```diff
@@ -49,12 +55,28 @@ export default {
...
     getApiRequest() {
-      this.getOrderBook({limit: 20});
+      this.loading = true;
+      this.getOrderBook({limit: 20})
+        .then(
+          () => {
+            this.loading = false;
+          }
+        )
+        .cath(
```
Comment (xerox-h20) [inline src/components/grid/OrderBook.vue]: typo in 'catch'

File: src/components/grid/Orders.vue [hunk 5]
```diff
@@ -97,7 +100,18 @@ export default {
...
     getApiRequest() {
-      this.getOrders();
+      this.loading = true;
+      this.getOrders()
+        .then(
+          () => {
+            this.loading = false;
+          }
+        )
+        .cath(
```
Comment (xerox-h20) [inline src/components/grid/Orders.vue]: typo in 'catch'

**Pull request merged**